ext/afform/core/afform.php

```diff
@@ -49,8 +48,8 @@ function afform_civicrm_config(&$config) {
   }
   Civi::$statics[__FUNCTION__] = 1;
 
-  Civi::dispatcher()->addListener(Submit::EVENT_NAME, [Submit::class, 'processContacts'], 500);
-  Civi::dispatcher()->addListener(Submit::EVENT_NAME, [Submit::class, 'processGenericEntity'], -1000);
+  Civi::dispatcher()->addListener('civi.afform.submit', ['\Civi\Api4\Action\Afform\Submit', 'processContacts'], 500);
```

Comment:
@ahed-compucorp  I am wondering if we should leave Submit::EVENT_NAME as it is as the issue is the class name that does not include name space?

Reply:
I tried to leave `Submit::EVENT_NAME`, but the same error was thrown again, but it work if I left `Submit::class`.

Reply:
Very interesting.   I'll leave this to you to decide and see what core team will decide.